### PR TITLE
feat: Add organization-wide scrapeJavaScript configuration

### PR DESCRIPTION
**Pull request by dcramer**
This adds an override for scrapeJavaScript to the organization settings. Its inverted from the existing functions (like requireDataScrubbers), and when set to false will disable scrapeJavaScript on all projects.

### CODE REVIEW

**Comment by dcramer** [comment]
screen shot 2018-07-05 at 2 09 05 pm
screen shot 2018-07-05 at 2 09 01 pm
screen shot 2018-07-05 at 2 17 08 pm

**Review by mattrobenolt — APPROVED** [review]
🎉 🎈

**Comment by dcramer** [comment]
@billyvg matt looked over backend, but would you mind glancing at the frontend code

File: src/sentry/static/sentry/app/data/forms/projectGeneralSettings.jsx [hunk 1]
```diff
@@ -195,6 +195,10 @@ export const fields = {
   scrapeJavaScript: {
     name: 'scrapeJavaScript',
     type: 'boolean',
+    disabled: ({organization, name}) => !organization[name],
```
Comment (billyvg) [inline src/sentry/static/sentry/app/data/forms/projectGeneralSettings.jsx]: Maybe add a comment saying if this turned OFF in org settings, then you cannot override in project settings.

**Review by billyvg — APPROVED** [review]

**Pull request merged**